lib/MusicBrainz/Server/Controller/Role/Load.pm

```diff
@@ -33,6 +35,8 @@ parameter 'allow_integer_ids' => (
     default => sub { 1 },
 );
 
+Readonly our $RELATIONSHIP_PAGE_SIZE => 100;
```

Comment:
My original suggestion was "Also, most relationship tables do not need to be paginated at all. I'd set a very high limit, 500 to 1000, if that is doable." - that said, maybe that's impractical and 100 is fine, especially now that I have a separate PR adding work filtering by name which effectively would allow me to replace my main requirement why I needed the composers' relationships tab to be easily searchable :) 

It would be nice to be able to filter rels eventually too, but it might be hard to do now :)

Reply:
I'd definitely want you to confirm that this doesn't break any use case before we merge it (so perhaps we should get your work filtering PR in at the same time). But I'd agree that filtering is the answer rather than outputting as much data as we can, 'cause that's precisely why our pages are so slow. I get that it's better to load more at once if the user is going to view all 20 pages anyway, but filtering is still the ideal solution for that.